pontoon/base/static/js/translate.js

```diff
@@ -2910,6 +2911,7 @@ var Pontoon = (function (my) {
           }
         },
         complete: function() {
+          self.XHRupdateOnServer = undefined;
```

Comment:
Setting to `undefined` is very weird and unusual to me. Why not set it to `null` instead?